### ISSUE TITLE
Question about Finetue MotionModel

### ISSUE THREAD

**Issue by LIRENDA621** [MooreThreads/Moore-AnimateAnyone#110]
请问二阶段加载animatediff的motion model进行finetune的时候，训练刚开始时模型去噪的结果是接近于噪音吗？我训练时是这个结果。但是如果不加载mm的预训练权重，由于零卷积的存在，训练刚开始时模型去噪的结果是正确的人像（也就是一阶段训练出的结果）

When loading animatediff's motion model for finetune in the second stage, is the model denoising result close to noise at the beginning of training? This is the result during my training. However, if the pre-training weights of mm are not loaded, due to the existence of zero convolution, the result of model denoising at the beginning of training will be the correct portrait (that is, the result of one-stage training)

**Comment by somuchtome** [comment]
> 请问二阶段加载animatediff的motion model进行finetune的时候，训练刚开始时模型去噪的结果是接近于噪音吗？我训练时是这个结果。但是如果不加载mm的预训练权重，由于零卷积的存在，训练刚开始时模型去噪的结果是正确的人像（也就是一阶段训练出的结果）
> 
> When loading animatediff's motion model for finetune in the second stage, is the model denoising result close to noise at the beginning of training? This is the result during my training. However, if the pre-training weights of mm are not loaded, due to the existence of zero convolution, the result of model denoising at the beginning of training will be the correct portrait (that is, the result of one-stage training)

 I've encountered a similar issue where there is pure noise at the beginning of fine-tuning a model. Have you figured out the reason behind this?

**Comment by supersonicMaclaurin** [comment]
Me too. got completelt noisy output and large loss which value around 1.0. The loss seems do not decreass.
